src/tasks/installers/operator.ts

```diff
@@ -243,103 +290,13 @@ export class OperatorTasks {
-          }
-        }
-      },
-      {
-        title: `Updating RoleBinding ${this.operatorRoleBinding} in namespace ${flags.chenamespace}`,
-        task: async (ctx: any, task: any) => {
-          const exist = await kube.roleBindingExist(this.operatorRoleBinding, flags.chenamespace)
-          const yamlFilePath = ctx.resourcesPath + 'role_binding.yaml'
-          if (exist) {
-            await kube.replaceRoleBindingFromFile(yamlFilePath, flags.chenamespace)
-            task.title = `${task.title}...updated.`
-          } else {
-            await kube.createRoleBindingFromFile(yamlFilePath, flags.chenamespace)
-            task.title = `${task.title}...created new one.`
-          }
-        }
-      },
-      {
-        title: `Updating ClusterRoleBinding ${clusterRoleBindingName}`,
-        task: async (_ctx: any, task: any) => {
-          const clusterRoleBindExists = await kube.clusterRoleBindingExist(clusterRoleBindingName)
-          const legacyClusterRoleBindExists = await kube.clusterRoleBindingExist(this.operatorClusterRoleBinding)
-          if (clusterRoleBindExists) {
-            await kube.replaceClusterRoleBinding(clusterRoleBindingName, this.operatorServiceAccount, flags.chenamespace, clusterRoleName)
-            task.title = `${task.title}...updated.`
-            // it is needed to check the legacy cluster object name to be compatible with previous installations
-          } else if (legacyClusterRoleBindExists) {
-            await kube.replaceClusterRoleBinding(this.operatorClusterRoleBinding, this.operatorServiceAccount, flags.chenamespace, this.operatorClusterRole)
-            task.title = `Updating ClusterRoleBinding ${this.operatorClusterRoleBinding}...updated.`
-          } else {
-            await kube.createClusterRoleBinding(clusterRoleBindingName, this.operatorServiceAccount, flags.chenamespace, clusterRoleName)
-            task.title = `${task.title}...created new one.`
-          }
-        }
-      },
-      {
-        title: `Updating ClusterRoleBinding ${operatorNamespaceEditorClusterRoleBindingName}`,
-        task: async (_ctx: any, task: any) => {
-          const clusterRoleBindExists = await kube.clusterRoleBindingExist(operatorNamespaceEditorClusterRoleBindingName)
-          if (clusterRoleBindExists) {
-            await kube.replaceClusterRoleBinding(operatorNamespaceEditorClusterRoleBindingName, this.operatorServiceAccount, flags.chenamespace, namespaceEditorClusterRoleName)
-            task.title = `${task.title}...updated.`
-          } else {
-            await kube.createClusterRoleBinding(operatorNamespaceEditorClusterRoleBindingName, this.operatorServiceAccount, flags.chenamespace, namespaceEditorClusterRoleName)
-            task.title = `${task.title}...created new one.`
-          }
-        }
-      },
+      this.getReadRolesAndBindingsTask(kube),
+      this.getCreateOrUpdateRolesAndBindingsTask(flags, 'Updating Roles and Bindings', false),
```

Comment:
Hmm... I thought here should be shouldUpdate = true: 

this.getCreateOrUpdateRolesAndBindingsTask(flags, 'Updating Roles and Bindings', true)

Reply:
You are right, there should be `true`.